osidb/tests/test_flaw.py

```diff
@@ -733,6 +740,58 @@ def test_validate_affect_ps_module_errors(self, bz_id, ps_module, should_raise):
             affect = AffectFactory(flaw=flaw, ps_module=ps_module)
             assert affect
 
+    @pytest.mark.parametrize(
+        "group,unembargo_date,should_raise",
+        [
+            (settings.PUBLIC_READ_GROUPS, None, True),
+            (settings.PUBLIC_READ_GROUPS, tzdatetime(2000, 1, 1), False),
+            (settings.EMBARGO_READ_GROUP, None, False),
+            (settings.EMBARGO_READ_GROUP, tzdatetime(2000, 1, 1), False),
```

Comment:
For testing the time stuff you can use `freezegun` like eg. at https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/tests/test_flaw.py#L411 so you can safely assume what is past/present/future and it will not change with the real time.